lib/vanagon/driver.rb

```diff
@@ -76,14 +79,13 @@ def run
       @engine.startup(@workdir)
 
       puts "Target is #{@engine.target}"
-
-      install_build_dependencies
+      retry_task { install_build_dependencies }
       @project.fetch_sources(@workdir)
       @project.make_makefile(@workdir)
       @project.make_bill_of_materials(@workdir)
       @project.generate_packaging_artifacts(@workdir)
       @engine.ship_workdir(@workdir)
-      @engine.dispatch("(cd #{@engine.remote_workdir}; #{@platform.make})")
+      retry_task { @engine.dispatch("(cd #{@engine.remote_workdir}; #{@platform.make})") }
       @engine.retrieve_built_artifact
```

Comment:
Should we retry this too in case something weird happens like a connection is accidentally dropped?